### ISSUE TITLE
Can't inherit "Customer:" from XmlReceipt

### ISSUE THREAD

**Issue by gustavovalverde** [odoo/odoo#18648]
Impacted versions:
10.0
Steps to reproduce:
1. Inherit XmlReceipt with `t-extend`
2.  Change the whole ticket part-by-part with `t-jquery` 

Current behavior:
![image](https://user-images.githubusercontent.com/16890942/28903360-7169e6d8-77d2-11e7-884e-7ceb4b29e895.png)
Just as this image shows, `Customer:`, which is shown in the printed receipt is not in the XML. I don't even know where this is coming from, so it can be inherited or changed.

Expected behavior:
There should be a code like:
```
                <div>--------------------------------</div>
                <t t-if='receipt.client'>
                    <div class='client'>
                        <div>Customer: <t t-esc='receipt.client' /></div>
                    </div>
```
After the cashier code, so this can be inherited and changed.

**Comment by gustavovalverde** [comment]
@jorenvo @Yenthe666, this is very similar to https://github.com/odoo/odoo/issues/17704 but it's strange as the code is not even there but it's showing in the printed receipt and I can't even find it 😅

**Comment by gustavovalverde** [comment]
Also those two `<br /><br />` should be inside something inheritable

**Comment by Yenthe666** [comment]
I guess  that `<br /><br />` could be moved up one line (so it falls within the div).
As about the customer, this is weird I guess? not really time to look further into that at the moment, sorry. 😉

**Comment by gustavovalverde** [comment]
Just to have a better graphical understanding. I've modified the whole ticket, but the Customer part can't be inherited or modified; it's not even inside the whole https://github.com/odoo/odoo/blob/10.0/addons/point_of_sale/static/src/xml/pos.xml
![customer](https://user-images.githubusercontent.com/16890942/28955856-58f2bfe0-78b7-11e7-8671-7b0e1dd5f31e.jpg)

**Comment by gustavovalverde** [comment]
@jorenvo, any updates here?

**Comment by jorenvo** [comment]
@gustavovalverde Are you running standard Odoo? I checked the value of `receipt` in `print_xml` in v10 via runbot but couldn't find a 'Customer:' line.

**Comment by gustavovalverde** [comment]
Yes, I'm running standard and couldn't find it, but it's getting printed
when using the PosBox.

**Comment by jorenvo** [comment]
I cannot reproduce your issue on v10 runbot. Could you give me exact steps you are following for it to print the 'Customer:' line?

**Comment by Yenthe666** [comment]
@gustavovalverde are you absolutely sure that nobody did custom development or that somebody altered the record in the database? I've never seen the customer line either & can't reproduce it either.
On a sidenote: as a neatpick the `<br/> <br/> ` could be moved in the div to improve things.

**Comment by torreslr** [comment]
Hi.. How you apply the width of the columns in the image ticket..??

**Comment by torreslr** [comment]
@gustavovalverde Hola, como pudiste dar un tamaño fijo a las columnas del orderline que se muestran en el ticket..??
